README.md

```diff
@@ -59,16 +104,12 @@ rdme openapi [path-to-file.json] --id={existing-id}
 
 #### Uploading or Editing an API Definition in a Project Version
 
-You can additional include a version flag, specifying the target version for your file's destination
+You can additional include a version flag, specifying the target version for your file's destination. This approach will provide you with CLI prompts, so we do not recommend this technique in CI environments.
 
 ```sh
 rdme openapi [path-to-file.json] --version={project-version}
 ```
 
-```sh
-rdme openapi [path-to-file.json] --id={existing-id} --version={project-version}
-```
-
```

Comment:
Just discovered this command which is incorrect since the `--id` flag implies the project version 🤦🏽